detection-rules/credential_phishing_esign_document_notification.yml

```
@@ -67,6 +67,8 @@ source: |
     or regex.icontains(body.html.raw, '>Docus[1l]gn<')
     // common greetings via email.local_part
     or any(recipients.to, strings.icontains(body.current_thread.text, .email.local_part))
+    // HR impersonation
+    or strings.ilike(sender.display_name, "H?R", "*Human Resources*")
```

`H?R` within strings.ilike _requires_ a character at between `H` and `R`.  Might be worth including `HR` has it's own string to make sure it matches without a separating character
```suggestion
    or strings.ilike(sender.display_name, "HR", "H?R", "*Human Resources*")
```

---

Whoops, good call, added